src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -22,27 +22,35 @@ export type Asset = AssetType;
 
 export type AssetSortType = SortType;
 
+export type OtherProps = {
+  teamId?: string;
+  initialAssetUrl?: string | null;
+  allowDeletion?: boolean;
+};
+
```

Comment:
OtherProps seems not to be imported separatedly, so you can combien them into one `type Props = {...}`